tests/conftest.py

```diff
@@ -34,6 +34,11 @@ def pytest_addoption(parser):
         action="store",
         help="SQLite DB URL used for tests with SpatiaLite4 (`sqlite:///path_to_db_file`).",
     )
+    parser.addoption(
+        "--sqlite_GeoPackage_dburl",
```

Comment:
this might be more logical as `"--sqlite_geopackage_dburl"`; the other command line options are also fully in lowercase, including terms like SQLite and PostgreSQL

Reply:
Indeed, thanks!